vcs/queries.py

```diff
@@ -123,31 +124,42 @@ def graphicsmethodtype(gobj):
     """
     Check the type of a graphics object.
 
-    Returns a None if the object is not a graphics method.
+    Returns None if the object is not a graphics method.
 
     :Example:
 
-::
-
-    a=vcs.init()
-    # Get an existing boxfill graphics method in VCS
-    box=a.getboxfill('quick')
-    # Get an existing isofill graphics method in VCS
-    iso=a.getisofill('quick')
-    # Get an existing line element in VCS
-    ln=a.getline('quick')
-    #...
-    # Will print 'boxfill'
-    print vcs.graphicsmethodtype(box)
-    # Will print 'isofill'
-    print vcs.graphicsmethodtype(iso)
-    # Will print None, because ln is not a graphics method
-    print vcs.graphicsmethodtype(ln)
-
-:returns: If gobj is a graphics method object, returns its type: 'boxfill', 'isofill', 'isoline',
-          'scatter', 'vector', 'xvsy', 'xyvsy', or 'yxvsx', 'taylordiagram'.
-          If gobj is not a graphics method object, returns None.
-:rtype: str or None
+        .. doctest:: queries_gmtype
+
+            >>> a=vcs.init()
+            ...
+
+            >>> # Get an existing boxfill graphics method in VCS:
+            >>> box=a.getboxfill('quick')
+            ...
+
+            >>> # Get an existing isofill graphics method in VCS
+            >>> iso=a.getisofill('quick')
+            ...
+
+            >>> # Get an existing line element in VCS
+            >>> ln=a.getline('red')
+            ...
+
+            >>> print vcs.graphicsmethodtype(box)
+            boxfill
+
+            >>> print vcs.graphicsmethodtype(iso)
+            isofill
+
+            >>> print vcs.graphicsmethodtype(ln)
+            Traceback (most recent call last):
+            ...
+            vcsError: The object passed is not a graphics method object.
+
+        :returns: If gobj is a graphics method object, returns its type: 'boxfill', 'isofill', 'isoline',
+                  'scatter', 'vector', 'xvsy', 'xyvsy', or 'yxvsx', 'taylordiagram'.
```

Comment:
@doutriaux1 , I submitted https://github.com/UV-CDAT/vcs/pull/7 to address the issues you have with this PR.